hacspec/src/math_integers.rs

```diff
@@ -325,7 +400,7 @@ macro_rules! unsigned_integer {
 
             /// `self ^ exp` where `exp` is a `u32`.
             #[cfg_attr(feature="use_attributes", library(hacspec))]
-            fn pow(self, exp: u32) -> Self {
+            fn exp(self, exp: u32) -> Self {
```

Comment:
Ibid, `usize` instead of `u32` ?

Reply:
The built-in `pow` uses `u32`... The std lib isn't really consistent here :/
https://doc.rust-lang.org/std/primitive.i16.html#method.pow

Reply:
Let's stick with std lib conventions then :)